.cargo/config.toml

```diff
@@ -6,3 +6,9 @@
 quick = "make quick"
 tidy = "make tidy"
 clippy-all = "clippy --all-features --all-targets -- -D warnings"
+
+# Use `cargo +nightly update` to update MSRV-aware (https://github.com/rust-lang/cargo/issues/13873)
+[unstable]
+msrv-policy = true
+[resolver]
+something-like-precedence = 'something-like-rust-version'
```

Comment:
I just find it remarkable that there must have been a real disagreement in the bikeshed discussion where they couldn't arrive at better names so called these options "something like X"